### PR TITLE
Fix homebrew input

### PR DESCRIPTION
**Pull request by emmauss**
I omitted writing to main layout during the refactor. which broke homebrew input. also add the joystick fix that got lost.

### CODE REVIEW

**Comment by minkcv** [comment]
Fixes the issue for me.

**Comment by emmauss** [comment]
rebased. should be good to merge

File: Ryujinx.HLE/Input/HidProController.cs [hunk 1]
```diff
@@ -35,10 +35,16 @@ public override void Connect(HidControllerId controllerId)
             HidJoystickPosition  rightStick)
         {
             long controllerOffset = WriteInput(buttons, leftStick, rightStick, HidControllerLayouts.ProController);
-
+
```
Comment (Thog) [inline Ryujinx.HLE/Input/HidProController.cs]: nit: white spaces

Comment (emmauss) [reply Ryujinx.HLE/Input/HidProController.cs]: done

**Comment by emmauss** [comment]
merging then

**Pull request merged**